src/decisionengine/framework/dataspace/datablock.py

```diff
@@ -278,18 +278,24 @@ def store_taskmanager(self, taskmanager_name, taskmanager_id):
     def get_taskmanager(self, taskmanager_name, taskmanager_id=None):
         """
         Retrieve TaskManager
+
         :type taskmanager_name: :obj:`string`
         :arg taskmanager_name: name of taskmanager to retrieve
         :type taskmanager_id: :obj:`string`
         :arg taskmanager_id: id of taskmanager to retrieve
-        :rtype: :obj: `dict`
+        :rtype: :obj:`dict`
```

Comment:
Since editing this could you use the Google format?
Or do you think mixed formats in the file would cause problems?